x-pack/packages/ml/inference_integration_flyout/components/flyout_layout.tsx

```diff
@@ -85,7 +88,7 @@ export const InferenceFlyout: React.FC<GenericInferenceFlyoutProps> = ({
             )}
             hasChildLabel={false}
             labelAppend={
-              <EuiLink href="TODO" external target={'_blank'}>
+              <EuiLink href={inferenceEndpointUrl} external target={'_blank'}>
                 <FormattedMessage
                   id="xpack.ml.inferenceFlyoutWrapper.elasticsearchModels.inferenceEndpointIdForm.inferenceEnpointDocumentation"
                   defaultMessage="What's this?"
```

Comment:
```suggestion
                  defaultMessage="Inference API documentation"
```

Reply:
Added in 7f55492a7417a15c75191a8dbf44a75bf623663a